docs/Sample Bulk Submission - Sheet1.tsv

```diff
@@ -0,0 +1,7 @@
+Sentence	Source
+Six years have passed since I resolved on my present undertaking.	Frankenstien, Mary Shelly, 1818, https://www.gutenberg.org/files/42324/42324-h/42324-h.htm 
+During her illness, many arguments had been urged to persuade my mother to refrain from attending upon her.	Frankenstien, Mary Shelly, 1818, https://www.gutenberg.org/files/42324/42324-h/42324-h.htm 
+She died calmly; and her countenance expressed affection even in death.	Frankenstien, Mary Shelly, 1818, https://www.gutenberg.org/files/42324/42324-h/42324-h.htm 
+My cat is a strange little dude.	Jessica Rose (self)
+I should have brought sunscreen.	Jessica Rose (self)
+Have you read the Doraemon comics yet?	Jessica Rose (self)
```

Comment:
Hi @jessicarose ! Would you mind adding a new line at the end of the file so this nasty little symbol at then disappears?